--- profiler.h	2021-01-15 15:05:18.000000000 +0100
+++ profiler.h	2022-02-21 15:15:49.247926000 +0100
@@ -17,6 +17,11 @@
 
 #include <cstdint>
 
+#ifdef DEFAULT
+#define DEFAULT_TEMP_VAL_PATCH DEFAULT
+#undef DEFAULT
+#endif
+
 namespace tflite {
 
 // A simple utility for enabling profiled event tracing in TensorFlow Lite.
@@ -191,4 +196,9 @@
     }                                                                      \
   } while (false);
 
+#ifdef DEFAULT_TEMP_VAL_PATCH
+#define DEFAULT DEFAULT_TEMP_VAL_PATCH
+#undef DEFAULT_TEMP_VAL_PATCH
+#endif
+
 #endif  // TENSORFLOW_LITE_CORE_API_PROFILER_H_
